addons/web/static/tests/webclient/actions/url_action_tests.js

```diff
@@ -48,4 +51,22 @@ QUnit.module("ActionManager", (hooks) => {
         await doAction(env, { type: "ir.actions.act_url" }, options);
         assert.verifySteps(["browser open", "onClose"]);
     });
+    
+    QUnit.test("execute an 'ir.actions.act_url' action with url javascript:", async (assert) => {
+        serviceRegistry.add(
```

Comment:
could you add `assert.expect(1)` as first line of the test? This will ensure that we enter your onRedirect mock

Reply:
it must be set at the beginning of the test